product-mini/platforms/windows/wasi_filtered_tests.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Open to suggestions on the naming/location of this file. Not sure if it fits here

Reply:
We don't have any socket tests disabled here although IIRC there's a bunch of functions not implemented. Can we make sure socket tests are actually being executed? Or does it mean we have very low test coverage for sockets?

Reply:
Thanks for spotting - I didn't realize we needed to compile the socket and thread tests to wasm in CI before executing them. Now we're building them via wasi-sdk and executing them. 

As expected, all (2) socket tests are failing so skipping them for now. Some of the thread tests which use `sleep` also rely on `poll_oneoff` being implemented so skipping those as well.